variables.tf

```diff
@@ -1,5 +1,100 @@
-variable "length" {
-  description = "The length of the random name"
-  type        = number
-  default     = 2
+variable "github_organization" {
+  type        = string
+  description = "The GitHub organization name"
+}
+
+variable "repository_collaborators" {
+  type = map(list(object({
+    username                    = string
```

Comment:
Given https://github.com/integrations/terraform-provider-github/pull/241, I wonder if we should add that `username` is case insensitive?

Reply:
See above.